tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/PathDescriptions.txt

```diff
@@ -0,0 +1,218 @@
+^//ldml/dates/calendars/calendar\[@type="[^"]*"]/monthPatterns/monthPatternContext\[@type="([^"]*)"]/monthPatternWidth\[@type="([^"]*)"]/monthPattern\[@type="([^"]*)"]; Provide the {1} and {2} version of the name for {3} month types. For more information, please see %DATE_TIME_NAMES_MONTH.
+^//ldml/localeDisplayNames/transformNames/transformName\[@type="([^"]*)"]; The name of the transform “{1}”. For more information, please see %TRANSFORMS_HELP.
+^//ldml/localeDisplayNames/codePatterns/codePattern[@type="([^"]*)"]; The pattern to be used when displaying a name for a character {1}. For more information, please see %LOCALE_PATTERN.
+^//ldml/localeDisplayNames/measurementSystemNames/measurementSystemName\[@type="([^"]*)"]; The name of the measurement system “{1}”. For more information, please see %UNITS_MISC_HELP.
+^//ldml/posix/messages/(no|yes)str; The word for “{1}”, lowercased, plus any abbreviations separated by a colon. For more information, see %UNITS_MISC_HELP.
+^//ldml/localeDisplayNames/annotationPatterns/annotationPattern[@type="([^"]*)"]; The pattern to be used when displaying a {1}. For more information, please see %LOCALE_PATTERN.
+^//ldml/characters/stopwords/stopwordList\[@type="([^"]*)"]; The words that should be ignored in sorting in your language. For more information, see %UNITS_MISC_HELP.
+^//ldml/dates/timeZoneNames/zone\[@type="([^"]*)"]/([^/]*)/(.*); Override for the $3-$2 timezone name for $1. For more information, see %TZ_CITY_NAMES.
+^//ldml/typographicNames/axisName[@type="([^"]*)"]; A label for a typographic design axis, such as “Width” or “Weight”. For more information, see %TYPOGRAPHIC_NAMES.
+^//ldml/typographicNames/styleName[@type="([^"]*)"][@subtype="([^"]*)"]; A label for a typographic style, such as “Narrow” or “Semibold”. For more information, see %TYPOGRAPHIC_NAMES.
+^//ldml/typographicNames/featureName[@type="([^"]*)"]; A label for a typographic feature, such as “Small Capitals”. For more information, see %TYPOGRAPHIC_NAMES.
+^//ldml/characterLabels/characterLabelPattern\[@type="([^"]*)"]\[@count="([^"]*)"]; A label for a set of characters that has a numeric placeholder, such as “1 Stroke”, “2 Strokes”. For more information, see %CHARACTER_LABELS.
+^//ldml/characterLabels/characterLabelPattern\[@type="([^"]*)"]; A modifier composed with a label for a set of characters. For more information, see %CHARACTER_LABELS.
+^//ldml/characterLabels/characterLabel\[@type="([^"]*)"]; A label for a set of characters. For more information, see %CHARACTER_LABELS.
+^//ldml/annotations/annotation\[@cp="([^"]*)"]\[@type="tts"]; A name for a character or sequence. For more information, see %SHORT_CHARACTER_NAMES.
+^//ldml/annotations/annotation\[@cp="([^"]*)"]; A set of keywords for a character or sequence. For more information, see %SHORT_CHARACTER_NAMES.
```

Comment:
In addition to avoiding duplication of the URLs, we could save some space and ensure consistency by adding macros for "For more information, see " (sometimes, inconsistently, with "please": "For more information, please see ") and "Note: before translating, be sure to read "